Rakefile

```diff
@@ -83,6 +83,7 @@ namespace :build do
       RakeUtils.system './deploy.sh'
     end
   end
+  task :'blockly-core' => :blockly_core
```

Comment:
Oh wait, I see now.

We probably also want one for `"code-studio"` then, yeah?